test/example-tests.sh

```diff
@@ -28,17 +28,17 @@ test_monolith() {
 test_monolith info
 test_monolith lint
 test_monolith deps
-test_monolith deps-of example/app-a
-test_monolith deps-on example/lib-a
-test_monolith with-all pprint :dependencies :source-paths
+test_monolith deps-of lein-monolith/example.app-a
+test_monolith deps-on lein-monolith/example.lib-a
+test_monolith with-all pprint :dependencies :source-paths :test-paths
 test_monolith each pprint :version
-test_monolith each :in lib-a pprint :root
-test_monolith each :upstream-of lib-b pprint :version
-test_monolith each :downstream-of lib-a pprint :name
+test_monolith each :in example.lib-a pprint :root :compile-path
+test_monolith each :upstream-of example.lib-b pprint :version
+test_monolith each :downstream-of example.lib-a pprint :name
```

Comment:
As an example of a benefit of the group naming suggestion, these lines would not need to change anymore.